.github/workflows/build.yml

```diff
@@ -101,7 +101,7 @@ jobs:
     if: github.event.action == 'closed' && github.event.pull_request.merged == true
     needs: release-prerequisites
 
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
Could you elaborate why `ubuntu-latest`?

Reply:
Yes. The plugin only works on linux and none of the other parts need to run on mac. Only vagrant builds/test need virtualbox/vagrant